### ISSUE TITLE
Make Row._data public

### ISSUE THREAD

**Issue by alextekartik** [simolus3/sqlite3.dart#125]
Hello Simon. When I don't use a cursor (PreparedStatement.select), I read a list of row data as a list (i.e. each row data is a list).

Currently if I use a cursor (PreparedStatement.selectCursor), I can only read a row and I have no way to access the row data as a List. I can only read them one by one by index or key.

To recreate the list I have to make this un-optimized call:
```dart
var data = List.generate(row.length, (index) => row[index]);
```
although I could simply do (if `_data` was named `data`)
```dart
var data = row.data
```
Would it be possible to make `_data` public in the Row class?

Thanks!

**Comment by simolus3** [comment]
I agree that this should be exposed. Maybe `Row.values` (a getter we have already inherited from `Map`) could be overriden to return a `List` which is just `data` internally.

**Comment by simolus3** [comment]
Updated in `sqlite3` version `1.9.1`.

**Comment by alextekartik** [comment]
Perfect! thanks. I implemented basic cursor (similar to your API, moveNext being async with some look-ahead buffer) with that (still published as dev for now) for sqflite_ffi. It even works on the web - with still some bugs though that i need to investigate.

Well done Simon!
